crates/cxx-qt-build/Cargo.toml

```diff
@@ -11,6 +11,7 @@ edition.workspace = true
 license.workspace = true
 description = "Helpers for integrating `cxx-qt` crate into a Cargo build"
 repository.workspace = true
+rust-version = "1.70.0"
```

Comment:
can you add a `clippy.toml` in the root folder.

In this you could specify the MSRV in clippy so that it doesn't warn for new clippy lints that aren't "available" yet
https://doc.rust-lang.org/stable/clippy/configuration.html#specifying-the-minimum-supported-rust-version

And as discussed before enable the `incompatible_msrv` lint too https://rust-lang.github.io/rust-clippy/master/#incompatible_msrv